test_helpers/pool_helper.go

```diff
@@ -192,7 +193,7 @@ func InsertOnInstances(servers []string, connOpts tarantool.Opts, space interfac
 }
 
 func SetInstanceRO(server string, connOpts tarantool.Opts, isReplica bool) error {
-	conn, err := tarantool.Connect(server, connOpts)
+	conn, err := tarantool.Connect(context.Background(), server, connOpts)
```

Comment:
We could add `ctx` as a first arg of the function.

Reply:
Updated.